osu.Game/Overlays/Settings/Sections/SkinSection.cs

```diff
@@ -34,13 +35,33 @@ public class SkinSection : SettingsSection
         private IBindable<WeakReference<SkinInfo>> managerAdded;
         private IBindable<WeakReference<SkinInfo>> managerRemoved;
 
+        private SettingsButton exportButton;
+
+        private Bindable<Skin> currentSkin;
+
         [BackgroundDependencyLoader]
         private void load(OsuConfigManager config)
         {
             FlowContent.Spacing = new Vector2(0, 5);
+
             Children = new Drawable[]
             {
                 skinDropdown = new SkinSettingsDropdown(),
+                exportButton = new SettingsButton
+                {
+                    Text = "Export selected skin",
+                    Action = () =>
+                    {
+                        try
+                        {
+                            skins.Export(skins.CurrentSkin.Value.SkinInfo);
+                        }
+                        catch (Exception)
+                        {
+                            Logger.Log("Could not export current skin", level: LogLevel.Error);
+                        }
```

Comment:
Should log the exception we have along the way?:
```suggestion
                        catch (Exception e)
                        {
                            Logger.Error(e, "Could not export selected skin!");
                        }
```

Reply:
Logger.Error is not meant for errors triggered by a user. It will report it as an error to us, which we do not want.

Reply:
Though you're logging it on an error-level (exactly like `Logger.Error(...)`), should change that to `Verbose` instead.

Reply:
No, `Verbose` is incorrect. It's an error, but one that shouldn't be logged to sentry.

If anything I'd be for logging the stack trace here for diagnostic purposes.